pkg/apic/client.go

```diff
@@ -88,8 +88,8 @@ type Client interface {
 	UpdateAccessControlList(acl *mv1a.AccessControlList) (*mv1a.AccessControlList, error)
 	CreateAccessControlList(acl *mv1a.AccessControlList) (*mv1a.AccessControlList, error)
 	UpdateAPIV1ResourceInstance(url string, ri *apiv1.ResourceInstance) (*apiv1.ResourceInstance, error)
-	CreateSubResourceScoped(scopeKindPlural, scopeName, resKindPlural, name, group, version string, subs map[string]interface{}) error
-	CreateSubResourceUnscoped(kindPlural, name, group, version string, subs map[string]interface{}) error
+	CreateSubResourceScoped(rm v1.ResourceMeta, subs map[string]interface{}) error
+	CreateSubResourceUnscoped(rm v1.ResourceMeta, subs map[string]interface{}) error
```

Comment:
Just making sure that the prev defs of these are not used anywhere outside the SDK

Reply:
I didn't use it in any agents, so I don't think so.